dev-python/pytest-forked/pytest-forked-1.0.2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} pypy{,3} )
+
+inherit distutils-r1
+
+DESCRIPTION="run tests in isolated forked subprocesses"
+HOMEPAGE="https://pypi.org/project/pytest-forked/ https://github.com/pytest-dev/pytest-forked"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-fbsd"
+IUSE="test"
+
+RDEPEND="
+	>=dev-python/pytest-2.6.0[${PYTHON_USEDEP}]"
+
+DEPEND="
+	${RDEPEND}
```

Comment:
You probably need `test? ( ${RDEPEND} )`

Reply:
Done.